homeassistant/components/knx/__init__.py

```diff
@@ -80,6 +80,9 @@
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.All(
+            # deprecated since 2021.3
+            cv.deprecated(CONF_KNX_CONFIG, replacement_key=CONF_KNX_CONFIG),
```

Comment:
```suggestion
```

I think we can take it out. It seems better to just have the message below instead of the wrong information.